pyensign/ensign.py

```diff
@@ -178,7 +178,9 @@ async def subscribe(
         self, *topics, on_event=ack_event, query="", consumer_group=None
     ):
         """
-        Subscribe to events from the Ensign server.
+        Subscribe to events from the Ensign server. This method returns immediately and
+        does not wait for events to be consumed from the topics. To implement a blocking
+        subscriber, callers can use `await asyncio.Future()`.
```

Comment:
Might need to add just a bit more user-oriented explanation here - does "returns immediately" mean that it essentially returns a cursor that users can use in subsequent calls to read events off the topic?